### PR TITLE
[WM-1658] match inputs/outputs for assemble_refbased to test workspace

### PR DESCRIPTION
**Pull request by mspector**

### CODE REVIEW

File: service/src/main/resources/changelog/changesets/20230125_assemble_refbased_io_updates.yaml [hunk 1]
```diff
@@ -0,0 +1,1417 @@
...
+                      },
+                      {
+                        "input_name": "assemble_refbased.reference_fasta",
+                        "input_type": {
+                          "type": "primitive",
+                          "primitive_type": "File"
+                        },
+                        "source": {
+                          "type": "record_lookup",
+                          "record_attribute": "workspace.ncov_reference_genome_fasta"
```
Comment (mspector) [inline service/src/main/resources/changelog/changesets/20230125_assemble_refbased_io_updates.yaml:37]: important: workspace data isn't implemented yet. what should we do here?

File: service/src/main/resources/changelog/changesets/20230125_assemble_refbased_io_updates.yaml [hunk 2]
```diff
@@ -0,0 +1,1417 @@
...
+                        "input_name": "assemble_refbased.plot_self_coverage.binning_summary_statistic",
+                        "input_type": {
+                          "type": "optional",
+                          "optional_type": {
+                            "type": "primitive",
+                            "primitive_type": "String"
+                          }
+                        },
+                        "source": {
+                          "type": "none"
```
Comment (kpierre13) [inline service/src/main/resources/changelog/changesets/20230125_assemble_refbased_io_updates.yaml:64]: Definitely not a blocker, but out of curiosity, does case matter for type 'none'?

**Review by kpierre13 — APPROVED** [review]

File: service/src/main/resources/changelog/changesets/20230125_assemble_refbased_io_updates.yaml (hunk 2 above)
Comment (mspector) [reply service/src/main/resources/changelog/changesets/20230125_assemble_refbased_io_updates.yaml:64]: I don't know - that's a good question. all lowercase is what we want, but maybe other cases would work too?

**Comment by mspector** [comment]
closing; redundant with https://github.com/DataBiosphere/cbas/pull/115

**Pull request closed without merging**